units/spawnhost_stop.go

```diff
@@ -50,6 +53,11 @@ func NewSpawnhostStopJob(h *host.Host, user, ts string) amboy.Job {
 	j.SetEnqueueAllScopes(true)
 	j.CloudHostModification.HostID = h.Id
 	j.CloudHostModification.UserID = user
+	j.UpdateRetryInfo(amboy.JobRetryOptions{
```

Comment:
Wrong PR :)

Reply:
nice catch.... between moving in to my new place and switching up on the tickets I didn't notice!